core/src/main/scala/com/rallyhealth/vapors/factfilter/dsl/ExprBuilder.scala

```diff
@@ -172,6 +173,16 @@ final class ValExprBuilder[V, R, P](returnOutput: Expr[Id, V, R, P])
   ): FoldOutExprBuilder[V, N, X, P] =
     new FoldOutExprBuilder(buildGetExpr(buildLens))
 
+  def value[X](
+    implicit
+    ev: R <:< TypedFact[X],
+    captureResult: CaptureResult[X],
+  ): ValExprBuilder[V, X, P] =
+    new ValExprBuilder(buildGetExpr[Id, X](_.field("value", _.value)))
```

Comment:
Why is the string `"value"` used here?

Reply:
This is because all `Fact`s have a `.value` field that contains the value of the `Fact`?

Reply:
I guess I'm still not clear why you are using a string literal in the expression builder.

Reply:
Yeah, I agree that you have a magic constant here that's not really connect to the `Fact` (when it really should....)